Resources/Prototypes/SS220/Maps/SS220_syndicate.yml

```
@@ -1,11 +1,12 @@
 - type: gameMap
-  id: ss220_NukieOutpost
+  id: SS220_syndicate
   mapName: Nukie Outpost
   mapPath: /Maps/NukeOps/ss220_Nuke_outpost_noshuttle.yml
   minPlayers: 0
   stations:
     SyndicateOutpost:
-      stationProto: StandardNukieOutpost
+      stationProto: StandardSS220_syndicate
       components:
         - type: StationNameSetup
-          mapNameTemplate: "Nukie Outpost"
+          mapNameTemplate: "SI-NO220 Heavy Hell "
```

```suggestion
          mapNameTemplate: "SI-NO220 Heavy Hell"
```
а мне больше Outer Heaven нравилось, так прикольно звучало...

---

Но как хочешь, конечно, я не настаиваю
тут просто пробел лишний